plugins/GSdx/GSTextureCache.h

```diff
@@ -152,7 +152,7 @@ class GSTextureCache
 		GSTexture* m_from_target;
 		GIFRegTEX0 m_layer_TEX0[7]; // Detect already loaded value
 		// Keep a GSTextureCache::m_map iterator to allow fast erase
-		std::array<uint16, MAX_PAGES> m_erase_it;
+		uint16 m_erase_it[MAX_PAGES];
```

Comment:
Well, personally I like the std::array. Features could still be useful in the future. It would be better to migrate `C` array to `C++` array instead. In order to avoid massive change for small gain, I prefer that new code uses them.